examples/python/src/hello-world.py

```diff
@@ -0,0 +1,199 @@
+import logging
+import os
+from posixpath import join
+
+import requests
+from flask import Flask, request
+
+logging.basicConfig(level=logging.DEBUG)
+
+app = Flask(__name__, static_url_path='/static', static_folder='./static')
+
+DEFAULT_APP_PORT = 8080
+DEFAULT_APP_HOST = 'localhost'
+DEFAULT_ROOT_URL = 'http://localhost:8080'
+FORM = {
```

Comment:
Not sure why this is capitalized. Even though it's only defined once, it's not really a "constant"

Reply:
replaced with lower case variable names